ui/app/components/app/account-menu/account-menu.component.js

```diff
@@ -16,19 +18,19 @@ import {
   CONNECT_HARDWARE_ROUTE,
   DEFAULT_ROUTE,
 } from '../../../helpers/constants/routes'
+import TextField from '../../ui/text-field'
 
-export default class AccountMenu extends PureComponent {
+export default class AccountMenu extends Component {
```

Comment:
I don't see any benefits of using `PureComponent` here, the container component is already pure.

Reply:
I don't think this is correct - the container component doesn't have any `shouldComponentUpdate` at all. What makes you think it's a pure component?

Reply:
connected component by default performs some additional checks to prevent re-rendering of the child (check `pure` flag https://react-redux.js.org/api/connect#pure-boolean and it's on by default)

Reply:
Ah, I didn't realize that was true by default. Makes sense.

Reply:
I also did not realize this. Neat.